device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/pg_profile_lookup.ini

```diff
@@ -1,17 +1,17 @@
 #  PG lossless profiles.
 # speed cable  size   xon    xoff  threshold
-  10000    5m 41984  19456  22528          0
-  25000    5m 41984  19456  22528          0
-  40000    5m 41984  19456  22528          0
-  50000    5m 41984  19456  22528          0
- 100000    5m 43008  19456  23552          0
-  10000   40m 41984  19456  22528          0
-  25000   40m 44032  19456  24576          0
-  40000   40m 45056  19456  25600          0
-  50000   40m 45056  19456  25600          0
- 100000   40m 49152  19456  29696          0
-  10000  300m 47104  19456  27648          0
-  25000  300m 56320  19456  36864          0
-  40000  300m 64512  19456  45056          0
-  50000  300m 69632  19456  50176          0
- 100000  300m 98304  19456  78848          0
+  10000    5m 19456  19456  22528          0
```

Comment:
We can use an unified Xoff value calculated based on the worst case, e.g., 100G and 2KM cables.